### PR TITLE
Move scrim on attraction detail carousel to be beneath paging dots

### LINKED ISSUE #112: Move scrim behind carousel dots on place detail screen
When a place detail screen has a carousel, the indicator is rendered behind the scrim:

image

This is in contrast to the home screen, where the indicator is rendered above the scrim.

I tried removing the place info (name, distance, and scrim) from the place detail screen, in hopes that the carousel component would automatically pick them up and render them (since the carousel layout includes them), but it didn't work.

### PR DESCRIPTION
**Pull request by maurizi**
## Demo
screenshot_20180724-114535

Closes #112

### CODE REVIEW

File: app/src/main/java/org/gophillygo/app/DetailCarouselViewListener.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    public DetailCarouselViewListener(Activity activity) {
+        this.inflater = activity.getLayoutInflater();
+    }
+
+    /**
+     * Get the destination to use as the data source for the carousel at the given position.
+     * @param position Scroll offset of the carousel
+     * @return Destination to display at that position
+     */
```
Comment (maurizi) [inline app/src/main/java/org/gophillygo/app/DetailCarouselViewListener.java]: This doc string is now out-of-date

File: app/src/main/res/layout/custom_home_carousel_item.xml [hunk 2]
```diff
@@ -32,7 +32,6 @@
             android:scaleType="centerCrop" />
 
         <android.support.constraint.ConstraintLayout
-            style="@style/ItemDetailInfo"
```
Comment (maurizi) [inline app/src/main/res/layout/custom_home_carousel_item.xml]: This shouldn't have been removed

Comment (flibbertigibbet) [reply app/src/main/res/layout/custom_home_carousel_item.xml]: I saw that, and thought it was moved to the new layout?

Comment (flibbertigibbet) [reply app/src/main/res/layout/custom_home_carousel_item.xml]: [here](https://github.com/azavea/cac-tripplanner-android/pull/150/files#diff-1c7694c2931e70d94de62af4124504f2R30)

Comment (maurizi) [reply app/src/main/res/layout/custom_home_carousel_item.xml]: That's for the detail activities - this layout is for the home activity, and shouldn't have been changed

**Pull request merged**